src/database/schema.ts

```diff
@@ -61,3 +75,74 @@ export const sessionRelations = relations(session, ({ one }) => ({
     references: [users.uuid],
   }),
 }))
+
+export const jobs = pgTable('jobs', {
+  ...createdAt,
+  uuid: uuid('uuid')
+    .default(sql`gen_random_uuid()`)
+    .primaryKey(),
+  user_uuid: uuid('user_uuid')
+    .notNull()
+    .references(() => users.uuid),
+  name: varchar('name'),
+  status: varchar('status').notNull(),
```

Comment:
for the status in the mvp text is fine but later on enum would be preferable - after you stabilized it